src/utils/validators.js

```diff
@@ -11,6 +11,7 @@ const fileNameRegexTest = /^[a-zA-Z0-9" "_-]+$/;
 const fileNameExtensionRegexTest = /^[a-zA-z]{3,4}$/;
 const RESOURCE_CATEGORY_REGEX = '^(([a-zA-Z0-9_-]+([\\s][a-zA-Z0-9_-]+)*)+)$';
 const resourceCategoryRegexTest = RegExp(RESOURCE_CATEGORY_REGEX);
+const resourceRoomNameRegexTest = /([a-zA-Z]+([-][a-zA-Z]+)?)+$/
```

Comment:
This regex doesn't quite work - it's not enforcing the start of the string, and also not allowing numerals, which conflicts with the error message.
I think we could use `^([a-zA-Z0-9]+-)*[a-zA-Z0-9]+$`, which is what I'm using in another PR to check for category/collection names

Reply:
fixed with [75f8df8](https://github.com/isomerpages/isomercms-frontend/pull/241/commits/75f8df811ed43f7e62a53ac7d7e1886fe132b4f2)